src/components/Books.js

```diff
@@ -3,6 +3,8 @@ import { useEffect } from 'react';
 import { fetchBooks } from '../redux/books/booksSlice';
 import Book from './Book';
 import AddNewBook from './AddNewBook';
+import books from '../style/books.module.css';
```

Comment:
- [ ] Great job working on the styles and modules but I think you have a typo here since it should be Books.module.css so that become with an error not found but after fixing it, it shows me the styles 🙏🏻🔥💪🏻✨ so please fix the typo to let your project work as expected 💪🏻🙏🏻


## You have the file name as Books.module.css not books.module.css 🤩

![image](https://github.com/astroboyReloaded/Bookstore/assets/58553711/b126444a-afd5-4b37-9abe-d5a94515850a)